drivers/setup/blue/blue.c

```diff
@@ -1073,16 +1075,28 @@ ScrIoControl(
 
             if (DeviceExtension->Enabled && DeviceExtension->VideoMemory)
             {
+#ifdef OPTIMIZED
+                CHAR attr = Buf->wAttribute;
```

Comment:
The real optimization is only here (caching global variable as a local one) and at other places like that.
Btw, did you test an optimized build? The difference will probably be way smaller

Reply:
> did you test an optimized build?

Yep, I did. I feel quicker.

Reply:
And what about an optimized build without your changes?

Reply:
The Release build is not tried yet.

Reply:
![image](https://user-images.githubusercontent.com/2107452/227063881-fe9d6668-5097-4a17-b1b6-edfdf333b8d8.png)
`error D8016` happened.

Reply:
Yes, add `-DRUNTIME_CHECKS=OFF` to the configure command line too.

Reply:
![image](https://user-images.githubusercontent.com/2107452/227066052-87b6a16c-fed8-4b60-acfe-adf1e13c775d.png)

Reply:
![image](https://user-images.githubusercontent.com/2107452/227067536-214f0a38-8b00-42c6-b2f6-05bde83bf5e6.png)
Fxxk again.

Reply:
![image](https://user-images.githubusercontent.com/2107452/227068690-6a44b078-caee-4bb5-8210-57d800a8cc0f.png)
Okay, okay.

Reply:
```txt
In optimized build:

BEFORE:
(base/setup/usetup/usetup.c:4471) CONSOLE_ClearScreen x 100: 39437376
(base/setup/usetup/usetup.c:4471) CONSOLE_ClearScreen x 100: 33289344
(base/setup/usetup/usetup.c:4471) CONSOLE_ClearScreen x 100: 34788864
(base/setup/usetup/usetup.c:4471) CONSOLE_ClearScreen x 100: 29840448

AFTER:
(base/setup/usetup/usetup.c:4471) CONSOLE_ClearScreen x 100: 27741120
(base/setup/usetup/usetup.c:4471) CONSOLE_ClearScreen x 100: 26541504
(base/setup/usetup/usetup.c:4471) CONSOLE_ClearScreen x 100: 29840448
(base/setup/usetup/usetup.c:4471) CONSOLE_ClearScreen x 100: 25491840

Comparison:
(39437376-27741120)/39437376 == 29.7%.
(33289344-26541504)/33289344 == 20.3%.
(34788864-29840448)/34788864 == 14.2%.
(29840448-25491840)/29840448 == 14.6%.
```